generate/templates/templates/binding.gyp

```diff
@@ -5,12 +5,10 @@
         ["OS=='win'", {
           "variables": {
             "is_electron%": "1",
-            "openssl_include_dir%": "<(module_root_dir)\\vendor\\openssl"
```

Comment:
Since we're cleaning up cruft, there's no reason to have a condition on `OS=='win'` anymore. We should remove that and just set `is_electron%` to `1` or `0`.